crypten/mpc/primitives/beaver.py

```diff
@@ -35,12 +36,33 @@ def __beaver_protocol(op, x, y, *args, **kwargs):
         x.size(), y.size(), op, device=x.device, *args, **kwargs
     )
 
-    # Vectorized reveal to reduce rounds of communication
     from .arithmetic import ArithmeticSharedTensor
 
-    eps_del = ArithmeticSharedTensor.reveal_batch([x - a, y - b])
-    epsilon = eps_del[0]
-    delta = eps_del[1]
+    if crypten.mpc.config.security == "active":
```

Comment:
Just for understandability, we may want to add a comment with reference material for the protocol we are implementing here. That would probably be helpful for this whole protocol actually.